src/v/config/configuration.cc

```diff
@@ -4134,6 +4134,17 @@ configuration::configuration()
        .example = "1.8",
        .visibility = visibility::tunable},
       2.0)
+  , datalake_disk_reservation_block_size(
```

Comment:
```
datalake_shard_disk_reservation_block_size
         ^^^^^
```
?

Reply:
?

Reply:
I believe adding `shard` to the name makes it slightly clearer what this configuration is for i.e. that this is the block size used by the per-shard disk/space management. Without the `shard` in the name it is not clear whether this is the reservation block size used by the translator/writer.